### DISCUSSION TITLE
Adjusting Stock Theme

### DISCUSSION THREAD

**Discussion by eric-hemasystems** [orbitalquark/textadept#260] [Q&A] [answered]
I'm mostly fine with the "light" theme that TextAdapt comes with but wish the background was a bit whiter. Is there a way to say "load 'light' theme but override just the background with color X"?

I tried adding something like this to my `init.lua` (slightly based on the [Themes](https://orbitalquark.github.io/textadept/manual.html#themes) section of the manual):

```lua
events.connect(events.LEXER_LOADED, function(name)
  view.style_back[view.STYLE_DEFAULT] = 0xFFFFFF
end)
```

It sort of worked. The background is now white but only where there is no code. Any code still has the dull gray color. Also would there be a way of using the colors defined in the inherited theme (i.e. `colors.light_white`) to be more semantic?

Or do I just need to copy the entire "light" stock theme into my own `~/.textadapt/themes` directory and adjust it?

**Comment by orbitalquark** [comment, marked as answer]
Unfortunately, the way theming is set up, you'd have to create your own light theme and use it. I'm actually working on a theme refresh right now and the next version will have a traditional white background. I also hope to improve theming such that these small tweaks are easier to make. Sorry for the inconvenience :(
